developer_tools/stratis_migrate_symlinks.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Migrate the symlinks of Stratis filesystems from the old
+# /stratis path to /dev/stratis, via synthetically generating
+# udev change events.
+
+if [ ! -d /stratis ]
+then
+	echo "'/stratis' directory does not exist; nothing to do."
+	exit 0
+fi
+
+for i in $(find /stratis)
```

Comment:
Can you decorate this loop with some docs about the rationale for this approach?